benchmarks/Riok.Mapperly.Benchmarks/MappingBenchmarks.cs

```diff
@@ -0,0 +1,118 @@
+﻿using BenchmarkDotNet.Attributes;
+using Riok.Mapperly.Benchmarks.Models;
+
+namespace Riok.Mapperly.Benchmarks;
+
+[MemoryDiagnoser]
+[InProcess]
+[HtmlExporter]
+[JsonExporterAttribute.Full]
+public class MappingBenchmarks
+{
+    private readonly Foo _foo;
+    private readonly Customer _customer;
+
+    public MappingBenchmarks()
```

Comment:
mb I thought you requested that I readd the mapster comparison benchmarks but without the benchmarks for the other libraries.  

> another Mapper around which would need updates with new features, etc.?

Im not sure what you mean by this, why would the current `MappingBenchmarks` need updates?

>  You mention a dll error, do you have more information on that? Probably I'll give it a try in the upcoming days.

I'll see if I can find my stash for this 👍. It would be nice to have a large complex mapper to track.

Reply:
> Im not sure what you mean by this, why would the current MappingBenchmarks need updates?

If a new feature is implemented (which should usually then also be in the benchmark and integration tests), `Foo`, `Customer`, `CustomerDto`, `MapperlyMapper` and `MappingBenchmarks` probably all need to be updated whereas if the integration test mappers could be reused changes would only have to be done once for a new feature.